tendrl/monitoring_integration/grafana/webhook_receiver.py

```diff
@@ -0,0 +1,52 @@
+import sys
+
+
+from gevent import socket
+
+
+from tendrl.commons.utils import log_utils as logger
+ 
+HOST = '127.0.0.1'
+PORT = 999
+
+
+def server():
```

Comment:
Remove all contents of this file. Use the gevent.wsgi as the http handler/server.  

eg: https://github.com/gevent/gevent/blob/master/examples/wsgiserver.py#L8  "PATH_INFO" == "/grafana_callback"